pkg/registries/azure/azure.go

```diff
@@ -22,7 +57,93 @@ func CreatorWithoutRepoList() (string, types.Creator) {
 	return types.AzureType,
 		func(integration *storage.ImageIntegration, options ...types.CreatorOption) (types.Registry, error) {
 			cfg := types.ApplyCreatorOptions(options...)
-			reg, err := docker.NewDockerRegistry(integration, true, cfg.GetMetricsHandler())
+			reg, err := newRegistry(integration, true, cfg.GetMetricsHandler())
 			return reg, err
 		}
 }
+
+func validate(cfg *storage.AzureConfig) error {
+	errorList := errorhelpers.NewErrorList("Azure container registry validation")
+	if cfg.GetEndpoint() == "" {
+		errorList.AddString("Endpoint must be specified for Azure container registry (e.g. <registry>.azurecr.io)")
```

Comment:
I think we should have these all start lowercase. There seems to be a mix in this PR, and I think lowercase is preferred for errors